src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor/Functions/LongRunningFunctions.cs

```diff
@@ -0,0 +1,25 @@
+using System.Diagnostics;
+using GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Model;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Logging;
+
+namespace GovUk.Education.ExploreEducationStatistics.Public.Data.Processor.Functions;
+
+public class LongRunningFunctions(ILogger<LongRunningFunctions> logger)
```

Comment:
Do the orchestration and trigger functions need to be in separate classes? Given we're just using this for testing, perhaps it'd be simpler to keep everything together in the same class?